manage/deploying/production/heroku_do.rst

```diff
@@ -0,0 +1,226 @@
+It supports many programming languages and necessary addons for different development environments.
+Deploying apps on Heroku is one of the way to show people our work samples without spending money.
+Plone has a `Heroku buildpack <https://plone.org/download>`_ that supports deploying automatically.
+On the other hand, this tutorial will show step-by-step deploying Plone app on Heroku manually for better understanding.
+
+Setting up Heroku account and local environment
+^^^^^^^^^
+
+First of all, to deploy an app on Heroku, we will need an account. 
+Head to `Heroku home page <https://www.heroku.com/>`_ to sign up for a free account and get started.
+
+.. image:: ../images/herokuapp.png
+    :align: center
+    :alt: Heroku sign up page
+
+We then need to install Heroku Toolbelt, which is a tool that helps deploying apps from our local machine to Heroku.
+You can find an installer that suits your computer with full instruction at `Heroku documentation <https://devcenter.heroku.com/articles/heroku-cli>`_.
+
+Download and deploy Plone
+^^^^^^^^^
+
+We are going to use the training-sandbox Plone package because it is designed for Heroku deployment.
+Firstly, we will clone the Plone training-sandbox.
+
+.. code-block:: shell
+
+    git clone https://github.com/collective/training-sandbox
+    $ cd training-sandbox
+
+We then need to create a Heroku app.
+
+.. code-block:: shell
+
+    heroku create --buildpack https://github.com/plone/heroku-buildpack-plone [your-app-name]
+    $ git remote -v
+        heroku	https://git.heroku.com/[your-app-name].git (fetch)
+        heroku	https://git.heroku.com/[your-app-name].git (push)
+
+Since we need database service for our Plone app, we need to have a database addon. 
+We will use Heroku Postgre. More information about it can be found `here <https://www.heroku.com/postgres>`_.
+
+.. code-block:: shell
+
+    heroku addons:create heroku-postgresql:hobby-dev --app [your-app-name]
+
+Now we should be good to go for deploying our app to Heroku
+
+.. code-block:: shell
+
+    git add .
+    $ git commit -m "your commit message"
+    $ git push heroku master
+
+
+Deploying your Plone app to Digital Ocean
+------------
+
+Digital Ocean is a service that provides VPS and root access for users, 
+which makes it very convenient since there are services that do not give full control over the VPS.
+We will use this platform for our tutorial since Digital Ocean is cheap and reliable.
+
+Setting up accounts and droplets
+^^^^^^^^^
+
+First of all, we will need to set up your Digital Ocean account at `Digital Ocean <https://www.digitalocean.com/>`_, 
+using our email and email password to sign up.
+
+After having your account ready, we will need to create a droplet.
+A droplet is the way Digital Ocean calls its own Virtual Private Server (VPS). 
+Click on the Create Droplet button on the right corner of the screen.
+
+.. image:: ../images/create_droplet.png
+    :align: center
+    :alt: Create droplet
+
```

Comment:
Wrong location for pictures, they should go into the _static under /root of the repo, 
lease follow our style-guide: https://docs.plone.org/about/rst-styleguide.html